src/blueapi/worker/reworker.py

```diff
@@ -70,6 +72,8 @@ def __init__(
         self._ctx = ctx
         self._stop_timeout = stop_timeout
 
+        self._pending_tasks = {}
```

Comment:
Are we going back on Blueapi not having more than 1 task queued?

Reply:
Done